introduction.md

```diff
@@ -4,3 +4,8 @@ Introduction
 Hi, i'm Jllavec and i'm mastering GitHub.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?